### PR TITLE
Rule discovery experimentation

### PR DESCRIPTION
**Pull request by heidmic**

### CODE REVIEW

File: logging_output_scripts/stat_analysis.py [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+                "level_0": ylabel,
+                0: xlabel
+            })
+
+            if metrics[metric] == "MSE":
+                add_prob = sample[sample[ylabel] == "ES"][xlabel] + sample[
+                    sample[ylabel] == "NSLC"][xlabel]
+                add_prob = pd.DataFrame({
+                    ylabel:
+                    np.repeat("ES $\wedge$ NSLC", len(add_prob)),
```
Comment (heidmic) [inline logging_output_scripts/stat_analysis.py:217]: this is AND but should be OR

File: logging_output_scripts/stat_analysis.py [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+                    xlabel:
+                    add_prob
+                })
+                sample = sample.append(add_prob)
+                add_prob = sample[sample[ylabel] == "ES"][xlabel] + sample[
+                    sample[ylabel] == "NSLC"][xlabel] + sample[sample[ylabel]
+                                                               == "NS"][xlabel]
+                add_prob = pd.DataFrame({
+                    ylabel:
+                    np.repeat("ES $\wedge$ NSLC $\wedge$ NS", len(add_prob)),
```
Comment (heidmic) [inline logging_output_scripts/stat_analysis.py:227]: this is AND but should be OR

File: logging_output_scripts/stat_analysis.py (hunk 1 above)
Comment (heidmic) [reply logging_output_scripts/stat_analysis.py:217]: https://github.com/dpaetzel/stat-analysis-rule-discovery-lcs/commit/c6cf8fd1b89b1a4238e8047028b66bbbf1856d5d

**Review by RomanSraj — APPROVED** [review]
LGTM

**Pull request merged**